executor/importer/import.go

```diff
@@ -300,8 +300,13 @@ func (e *LoadDataController) initFieldParams(plan *plannercore.LoadData) error {
 	return nil
 }
 
+var ignoreInTest = false
```

Comment:
I suggest that it is an atomic type to avoid data race report.

Reply:
will turn on race check in bazel in next commit

Reply:
https://github.com/pingcap/tidb/pull/42667/commits/8827c5d0c2c0c2c5b4259401241cf62b56a5c9eb